apps/prairielearn/src/ee/pages/institutionAdminSso/institutionAdminSso.ts

```diff
@@ -1,6 +1,6 @@
 import { Router } from 'express';
 import asyncHandler = require('express-async-handler');
-import z from 'zod';
+import { z } from 'zod';
```

Comment:
Does this need to be done elsewhere? Here's another instance where this may be needed:
https://github.com/PrairieLearn/PrairieLearn/blob/164b5d5db2cc218fda863909971dfd27389093ba/apps/prairielearn/src/lib/assessment.js#L8
https://github.com/PrairieLearn/PrairieLearn/blob/164b5d5db2cc218fda863909971dfd27389093ba/apps/prairielearn/src/pages/administratorWorkspaces/administratorWorkspaces.html.js#L3

Reply:
Done in 977b524!